### ISSUE TITLE
Django 1.8: AttributeError: 'Options' object has no attribute 'module_name'

### ISSUE THREAD

**Issue by nkonin** [django-extensions/django-extensions#674]
```python
AttributeError at /

'Options' object has no attribute 'module_name'

Request Method: 	GET
Request URL: 	http://127.0.0.1:8000/
Django Version: 	1.8.1
Exception Type: 	AttributeError
Exception Value: 	

'Options' object has no attribute 'module_name'

Exception Location: 	/home/nkonin/work/test/testvenv/lib/python3.4/site-packages/django_extensions/admin/__init__.py in get_urls, line 63
Python Executable: 	/home/nkonin/work/test/testvenv/bin/python
```

See: https://docs.djangoproject.com/en/1.8/releases/1.8/#model-meta-api

**Comment by lsemel** [comment]
Getting this also

**Comment by trbs** [comment]
This should be fixed in the latest release.

**Comment by bolshoibooze** [comment]
Doesn't look like it has been fixed, just started a project with Django-1.8.3 and the same issue persists

**Comment by trbs** [comment]
@bolshoibooze what is the exact error / traceback and which version of django extensions are you using ?
